.github/workflows/clerk-user-cleanup.yml

```diff
@@ -0,0 +1,46 @@
+name: Clerk User Cleanup
+
+on:
+  schedule:
+    # Run daily at 2 AM UTC on staging
+    - cron: "0 2 * * *"
+  workflow_dispatch:
+    inputs:
+      dry_run:
+        description: "Run in dry-run mode (no actual deletions)"
+        required: true
+        default: true
+        type: boolean
+
+permissions: {}
+
+jobs:
+  clerk-user-cleanup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run Clerk User Cleanup on Staging
+        run: |
+          DRY_RUN="${{ github.event.inputs.dry_run || 'false' }}"
```

Comment:
is 'false' here a string? Wasn't sure whether it's casted at some point

Reply:
I don't know the exact answer of how these are interpolated, but it does seem to work as expected. It's a good question

Reply:
DRY_RUN is ultimately only passed forward into the URL concatenation, so I suppose it doesn't matter what the type is here in this case